pkgs/servers/irc/ergochat/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildGo117Module, fetchFromGitHub, lib, nixosTests }:
+
+buildGo117Module rec {
+  pname = "ergo";
```

Comment:
unsure if this should be `ergochat`?

Reply:
No. it's ergo if you see the README.

Reply:
`ergochat` is only the organization on GitHub. 
Documentation, project_name and logo are bound to `ergo`, ergo the `pname` should be called `ergo`.

Reply:
alright, just thought that maybe `nix-env -i` would be conused then (but who uses that anyway)

Reply:
Yeah. does not make sense for an IRC server...